tools/src/main/java/org/apache/kafka/trogdor/common/Platform.java

```diff
@@ -54,6 +62,29 @@ public static Platform parse(String curNodeName, String path) throws Exception {
         }
     }
 
+    class MetricsContainer {
+        private static TrogdorMetrics trogdorMetrics = null;
```

Comment:
I'm a bit confused by why this is static.  It seems like it should just be tracked by each Coordinator or Agent.